### ISSUE TITLE
Variable max subscription empty when using default archive settings

### ISSUE THREAD

**Issue by Kahmoon** [codefog/contao-events_subscriptions#17]
Hi,

i´m using some of your variables in template to provide more information for our students. 

`<p><strong>Max. Teilnehmer:</strong> <?= $this->subscription_maximum ?></p>`

Default archive settings:

- max subscriptions: 7

When i do **not** overwrite subscription settings in an event itself this variable is 0. 
When i use overwrite subscription settings, the correct number 7 is shown.

Frank

**Comment by qzminski** [comment]
Before I fix it please try the following solution:

```php
<?= $this->subscription_override ? $this->subscription_maximum : $this->calendar->subscription_maximum ?>
```

**Comment by Kahmoon** [comment]
Still no value 

`<p><strong>Max. Teilnehmer:</strong> <?= $this->subscription_override ? $this->subscription_maximum : $this->calendar->subscription_maximum ?></p>`

**Comment by qzminski** [comment]
What template is it exactly? Did you check what `print_r($this->calendar)` outputs?

**Comment by Kahmoon** [comment]
Its the event_full_registration.html5

Debug in template:
`<!--# archive: <?= $this->calendar->subscription_maximum ?># event: <?= $this->subscription_maximum ?>-->`
Output:
`<!--# archive: # event: 0-->`



print_r($this->calendar) does not output anything. <?php dump($this->getData()) ?> does output

 "subscription_waitingListLimit" => "0"
  "subscription_waitingList" => ""
  "subscription_unsubscribeEndTime" => ""
  "subscription_subscribeEndTime" => ""
  "subscription_maximum" => "0"
  "subscription_types" => null
  "subscription_override" => ""

**Comment by Kahmoon** [comment]
Do i use the wrong template or is it just a bug? :-)

**Comment by qzminski** [comment]
I don't know yet, will let you know when I'm on it :+1:

**Comment by qzminski** [comment]
Fixed in 12838e5.

**Comment by qzminski** [comment]
Once this fix is merged, you should be able to use the template variables that are described in this method: https://github.com/codefog/contao-events_subscriptions/blob/master/src/Codefog/EventsSubscriptions/FrontendModule/SubscriptionTrait.php#L33

So for example:

```php
Maximum subscriptions: <?= $this-> subscriptionMaximum ?>
```
